integreat_cms/cms/templates/translations/translations_management.html

```diff
@@ -34,9 +34,30 @@ <h3 class="heading font-bold text-black">
                     </div>
                 </div>
             </div>
+            <div class="rounded border border-solid border-blue-500 shadow-2xl bg-white">
+                <div class="p-4 rounded bg-water-500">
+                    <h3 class="heading font-bold text-black">
+                        <i icon-name="pie-chart" class="mr-2"></i> {% translate "Translation Budget" %}
+                    </h3>
+                </div>
+                <div class="px-4 pb-4 divide-y divide-gray-200 space-y-4">
+                    <div>
+                        <div id="budget-graph-container">
+                            <canvas id="budget-graph"
+                                    data-budget-used="{{ region_budget.used|intcomma }}"
+                                    data-budget-total="{{ region_budget.total|intcomma }}"></canvas>
+                        </div>
+                        <div class="text-center font-bold">
+                            <p>{% translate "Available credits" %}</p>
+                            <p>{{ region_budget.remaining|intcomma }} {% translate "words" %}</p>
+                        </div>
+                        <p>
+                            {{ region_budget.used|intcomma }} {% translate "of" %} {{ region_budget.total|intcomma }} {% translate "words used so far." %}
```

Comment:
```suggestion
                            {% blocktranslate trimmed with used=request.region.deepl_budget_used|intcomma total=request.region.deepl_budget|intcomma %}
                                {{ used }} of {{ total }} words used so far
                            {% endblocktranslate %}
```
When ever you insert text fragments, I think it makes sense to keep the context for translations. Even if the word order is identical in German here, we might translate the CMS into additional languages in the future where the word order might differ.